src/main/java/io/github/thebusybiscuit/hotbarpets/HotbarPet.java

```diff
@@ -19,4 +29,26 @@ public HotbarPet(Category category, SlimefunItemStack item, ItemStack food, Item
 	public ItemStack getFavouriteFood() {
 		return this.food;
 	}
+
+	/**
+	 * Tries to consume the pets favourite food, if this was successful it will return true. Otherwise false.
+	 * If it fails to consume food it will send a message to the player, this is on a delay as to not spam the player.
+	 * The default delay is {@link #MESSAGE_DELAY}.
+	 *
+	 * @param player  The {@link Player} who owns this pet
+	 * @return If the food consumption was successful
+	 */
+	public boolean tryToConsumeFood(Player player) {
```

Comment:
Don't really like the way this method sounds..
"isFed"?

Reply:
it doesnt check if they are fed though, it tries to feed can change it tryToFeed